cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -81,16 +85,26 @@ def test_embedded_ansible_event_catcher_process(embedded_appliance):
         initialEstimate: 1/4h
         tags: ansible_embed
     """
-    result = embedded_appliance.ssh_client.run_rake_command(
-        "evm:status | grep 'EmbeddedAnsible'"
-    ).output
+    if embedded_appliance.version < "5.11":
+        result = embedded_appliance.ssh_client.run_rake_command(
+            "evm:status | grep 'EmbeddedAnsible'"
+        ).output
+
+        for data in result.splitlines():
+            logger.info("Checking service/process %s started or not", data)
+            assert "started" in data
+    else:
+        rpm_check = embedded_appliance.ssh_client.run_command(
+            "rpm -qa | grep 'ansible-runner'"
+        ).output
 
-    for data in result.splitlines():
-        logger.info("Checking service/process %s started or not", data)
-        assert "started" in data
+        for data in rpm_check.splitlines():
+            logger.info("Checking %s is present or not", data)
```

Comment:
**Optional:** Using f-string instead of this formatting.

Reply:
Updated.